plugins/crashes/frontend/public/templates/overview.html

```diff
@@ -1,25 +1,34 @@
 <div>
     <cly-tabs v-model="currentTab">
```

Comment:
cly-tabs should be replaced with cly-dynamic-tabs, whenever time permits
just leaving a note here for reference.